variables.tf

```diff
@@ -353,6 +353,12 @@ DESCRIPTION
   nullable    = false
 }
 
+variable "private_link_excluded_zones" {
+  default = ["privatelink.pbidedicated.windows.net", "privatelink.analysis.windows.net", "privatelink.tip1.powerquery.microsoft.com"]
+  type = list(string)
```

Comment:
nit pick: do we think this should be a set as only will be unique values? does mean youll need to do a `tolist()` in the locals though

Reply:
the `tolist` doesn't seem to be needed, as `contains` works with sets as well: https://developer.hashicorp.com/terraform/language/functions/contains